src/garage/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Garage Base."""
-from garage._dtypes import InOutSpec
+from garage._dtypes import ModuleInOutSpec
```

Comment:
why the name change?

Reply:
i would prefer if you kept the old name -- nothing says this type can only be used by Modules